pyquil/pyqvm.py

```diff
@@ -281,31 +226,20 @@ def write_memory(self, *, region_name: str, offset: int = 0, value=None):
 
     def run(self):
         self.status = 'running'
-        assert self._qubit_to_ram is not None
-        assert self._ro_size is not None
+        p = self.program
```

Comment:
I think `p` is only referenced one place below. Not that there is anything wrong with that (especially since it's inside a loop). Just pointing it out in case this a vestige of some intermediate refactoring where `p` appeared multiple times.